README.md

```diff
@@ -90,15 +113,15 @@ This module should be built as an OSGi bundle, so that it can more easily be add
 The API is under review, particularly to make it easier to use with YAML blueprints.
```

Comment:
A couple of lines above:
```
 This module should be built as an OSGi bundle, so that it can more easily be added to Brooklyn.
```
This should already be the case.